sonic-net/sonic-buildimage
# [Mellanox] Enable CMIS host mgmt. feature to Mellanox-SN4700-O8C48, Mellanox-SN4700-O8V48 SKUs

#### Why I did it
To enable CMIS host mgmt feature
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
N/A
#### How to verify it
Load the system with the relevant SKU, verify CMIS host management is enabled, and per supported module check SW control is on.


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
